### DISCUSSION TITLE
[Feature] Show and filter assets via media types

### DISCUSSION THREAD

**Discussion by chrneu** [immich-app/immich#8544] [Feature Request]
### The feature

Like a collection of albums, it would be nice to automatically categorize assets into specific types when a photo was added.

Types could be:
- Video (file type)
- RAW (file type)
- Panorama (resolution)
- Screenshot (ml, exif?)
- Landscapes (ml)
- Group photos (ml face detection)
- Selfies / Portaits (ml face detection / exif?)
- Bursts (timestamp)
- Live Photos (file type)
- Time-lapse (exif)
- Slo-mo (exif)
- Duplicates (hash, ml to detect similar photos and do a ranking like https://utorontomist.medium.com/photoml-photo-culling-with-machine-learning-908743e9d0cb)
- ...

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Does this overlap with #1673?

**Reply by chrneu** [reply]
Somehow related, but from my understanding a smart album is something where a user wants to create an album adhoc, with all the search options which are available, like "Show me all photos from Person within Timeframe at Location". 

While this is a standard set provided by immich, showing standard / extended types of the available content.

**Reply by bo0tzz** [reply]
> a smart album is something where a user wants to create an album adhoc

Not quite, the idea is that the rules are evaluated every time you upload an asset and such.

**Reply by chrneu** [reply]
Yes, you are right. I meant with it that the creation is based on a user decision, while this comes as out of the box. But of course, a user could just create it as a smart rule set, even though I would appreciate it to have a standard set of smart rules.

**Reply by AngelaDMerkel** [reply]
I think this is the best approach. Allowing there to be typologies of images (ie Panorama is 2:1) would allow users to have automatically populated folders for specific kinds of images and alleviate the need to have Immich decide what kinds of images are most relevant to users. I don't know the extent of the effort involved, but being able to search "Portrait" and having images with a certain percent of the frame occupied by the face would be very useful

**Reply by DavidMBK03** [reply]
Is there any update on whether this is coming soon or not?
